oci/spec_opts.go

```diff
@@ -894,9 +894,11 @@ func WithAppendAdditionalGroups(groups ...string) SpecOpts {
 			if err != nil {
 				return err
 			}
-			ugroups, err := user.ParseGroupFile(gpath)
-			if err != nil {
-				return err
+			var ugroups []user.Group
+			var groupErr error
+			ugroups, groupErr = user.ParseGroupFile(gpath)
+			if groupErr != nil && !os.IsNotExist(groupErr) {
```

Comment:
Not a show-stopper, but just realised we don't really need the `var` declarations here, and could keep the original;

```suggestion
			ugroups, groupErr := user.ParseGroupFile(gpath)
			if groupErr != nil && !os.IsNotExist(groupErr) {
```

Reply:
Done